src/Template/Template.styled.tsx

```diff
@@ -0,0 +1,35 @@
+import { css, useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import Template from './Template.component';
+
+export default (props) => (
+  <Template
+    css={(theme) =>
+      css`
+        color: ${theme.uikit.colors.grey};
+        border: 20px solid gold;
+      `
+    }
+    {...props}
+  />
+);
+
+export const StyledTemplate = styled(Template)`
+  border: 10px green solid;
+  color: ${({ theme }) => theme.uikit.colors.grey};
+`;
+
+// not quite idiomatic use but emotion theme has usage like this in projects
```

Comment:
there's two reasons why we do this in other projects (e.g. Arranger):
- sometimes we pass the `theme` into functions that generate objects to be passed as props, some of which may be styled components themselves.
- doing `css={theme =>...` in every component breaks DRY, seems to me. That pattern is useful in places where we talk about local instances of an object (e.g. `prev` in `useState`), but here the theme is shared across components as if it was a constant rather than as an instance of itself. Thus, "import" the `theme` once (or "define" it? whatever `useTheme` does lol), and use it many times.